submissions/TatyanaMolchanova/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,174 @@
+                                             Blogs
+                                         </a>
+                                     </div>
+                                     <div class="popup-item">
+                                        <a href="#" class="popup-link">
+                                             <img src="./images/calendar.png" alt>
+                                             Calendar
+                                         </a>
+                                     </div>
+                                     <div class="popup-item">
+                                        <a href="#" class="popup-link">
+                                             <img src="./images/classroom.png" alt>
+                                             Class
+                                         </a>
+                                     </div>
+                                     <div class="popup-item">
+                                        <a href="#" class="popup-link">
+                                             <img src="./images/files.png" alt>
+                                             Files
+                                         </a>
+                                     </div>
+                                     <div class="popup-item">
+                                        <a href="#" class="popup-link">
+                                             <img src="./images/mail.png" alt>
+                                             Mail
+                                         </a>
+                                     </div>
+                                     <div class="popup-item">
+                                        <a href="#" class="popup-link">
+                                             <img src="./images/maps.png" alt>
+                                             Maps
+                                         </a>
+                                     </div>
+                                     <div class="popup-item">
+                                        <a href="#" class="popup-link">
+                                             <img src="./images/mars.png" alt>
+                                             Mars
+                                         </a>
+                                     </div>
+                                </div>
+                            </div>
+                          </div>
+                    </div>
+                </div>
+
+            </li>
+            <li class="nav-item">
+                <a href="#" class="nav-link nav-image bell">
+                    <img src="./images/bell.png" alt="Alerts">
+                </a>
+            </li>
+            <li class="nav-item">
+                <a href="#" class="nav-link nav-image profile">
+                    <img src="./images/contacts.png" alt="Prfile">
+                </a>
+            </li>
+        </ul>    
+    </nav>
+
+
+    
+
+</body>
+</html>
```

Comment:
Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).

Reply:
I added empty lines to .html and .css files. Thank you!